spec/unit/space_spec.rb

```diff
@@ -0,0 +1,14 @@
+require './app/model/space.rb'
```

Comment:
this should be already available with the following require chain: spec_helper < app.rb < data_mapper_setup.rb < model/space.rb